fetch-api/fetch-referrer-policy.js

```diff
@@ -0,0 +1,9 @@
+var baseUrl = 'users.json';
+var policies = ['no-referrer', 'no-referrer-when-downgrade', 'origin',
+  'origin-when-cross-origin', 'unsafe-url'];
+
+policies.forEach(policy => {
+  var url = `${baseUrl}?policy=${policy}`;
```

Comment:
It is confusing as it looks like we can pass `policy` in the URL which is not the case.
How about something like this below?

```js

var policies = ['no-referrer', 'no-referrer-when-downgrade', 'origin',
    'origin-when-cross-origin', 'unsafe-url'];

policies.forEach(policy => {
  fetch('users.json', {referrerPolicy: policy})
  .then(response => { 
    ChromeSamples.log(`Fetched URL with ${policy}`);
  });
});